### ISSUE TITLE
Publikování na WordPress.org

### ISSUE THREAD

**Issue by kalich5** [skaut/dsw-skaut-svs#7]
Aby mohla být šablona publikována, musí se řídit https://developer.wordpress.org/themes/

**Comment by vojta001** [comment]
Má to význam? Šablona je určena pro velmi specifické publikum a budou-li automatické aktualizace z GitHubu (#1), nevidím k tomu důvod

**Comment by kalich5** [comment]
Zatím to na pořadu dne není, ale časem to smysl mít bude si myslím. ;-) Teď hlavně doladit ty aktualizace, aby to fungovalo :-)

**Comment by kalich5** [comment]
myslím, že tohle je jedna z hlavních priorit. Aby aktualizace prostě fungovala všem

**Comment by vojta001** [comment]
A ta z Githubu nějak stávkuje?

**Comment by kalich5** [comment]
jo, na MultiSite to nefunguje a automatická aktualizace taky nejede. Musíš na to klikat. Není to prostě ono
